src/e3sm_io.c

```diff
@@ -104,6 +104,8 @@ static void usage (char *argv0) {
        [-g num] Number of subfiles, used by Log-based VOL and ADIOS I/O only,\n\
                 -1 for one subfile per compute node, 0 to disable subfiling,\n\
                 (default: 0).\n\
+       [-t time] Simulated computation time (sec) of a timestep (default: 0).\n\
+                 (Does not count toward end-to-end time)\n\
```

Comment:
Why is it not counted as end-to-end time?
We can have a separate timer for I/O only.

A better description is "Add sleep time to emulate the computation in order to overlapping I/O when Async VOL is used."

Reply:
Please update INSTALL.md for this new command-line option.

Reply:
> 

It is to not distort the end to end bandwidth.
I/O time is flush only. end to end time includes other file metadata.